composer.json

```diff
@@ -41,6 +47,7 @@
         ],
         "check:php:php-cs-fixer": "vendor/bin/php-cs-fixer check",
         "check:php:phpstan": "vendor/bin/phpstan --configuration=phpstan.neon",
+        "check:integration": "vendor/bin/phpunit --testsuite=integration",
```

Comment:
I would prefer `tests:integration`.

Reply:
As the others are all verbs, how about test:integration

Reply:
Go for it